### PR TITLE
Allow to start qemu later

### PR DESCRIPTION
**Pull request by nadvornik**
This will be used as a replacement for waiting in main.pm.

### CODE REVIEW

File: testapi.pm [hunk 1]
```diff
@@ -1413,6 +1413,22 @@ sub freeze_vm {
     query_isotovideo('backend_freeze_vm');
 }
 
+=head2 cont_vm
```
Comment (coolo) [inline testapi.pm]: as we're dragging this into public API, can we please fix the function name to be english?

Comment (nadvornik) [inline testapi.pm]: continue_vm ? Or do you have any better suggestion?

Comment (coolo) [inline testapi.pm]: what is the opposite of freeze? continue is at least part of http://www.wordhippo.com/what-is/the-opposite-of/freeze.html

But I like melt better :)

But resume_vm has its appeal too

**Comment by nadvornik** [comment]
OK, I renamed it to resume_vm

**Comment by okurz** [comment]
LGTM

**Pull request merged**